MANIFEST.in

```
@@ -1 +1 @@
-recursive-include wgtools/frontend/templates *
+recursive-include wgke/frontend/templates *
```

kex is a more common abbreviation of key exchange.

```suggestion
recursive-include wgkex/frontend/templates *
```

---

Thanks, I've used wgkex in the current draft.